examples/with_pipeline.py

```diff
@@ -0,0 +1,40 @@
+#  Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
how about naming iris_with_pipeline? in case there will be more examples of pipelines